setup.py

```diff
@@ -36,8 +36,15 @@
 tensorboard_extra_require = ["tensorflow >=2.3.0, <=2.5.0"]
 metadata_extra_require = ["pandas >= 1.0.0"]
 xai_extra_require = ["tensorflow >=2.3.0, <=2.5.0"]
+profiler_extra_require = ["tensorboard-plugin-profile", "tensorflow >=2.4.0"]
+
 full_extra_require = list(
-    set(tensorboard_extra_require + metadata_extra_require + xai_extra_require)
+    set(
```

Comment:
TF version should be handled explicitly since TB, XAI, and Profiler have different version bounds.

Reply:
Done.